.github/workflows/close-inactive-issues.yaml

```diff
@@ -0,0 +1,26 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 18 * * *" # 10:30am PT
+  workflow_dispatch: # Allows for manual triggering if needed
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      # See https://github.com/actions/stale for
+      # details on the parameters for this action
+      - uses: actions/stale@v7
+        with:
+          debug-only: true # TODO: Delete this line when we are ready to implement this for our process.
+          stale-issue-label: "waiting for customer response"
```

Comment:
@jacob314 @kenzieschmoll 
This still needs to be tested, but hypothetically these settings will result in the following behaviour:
- user submits issue
- we ask for more info
- we mark it with "waiting for response"
- the workflow picks up the issue as something to track, since it has the label.
- if the user responds within 14 days the the label would be removed
- if they don't then the issue will be closed with the message

Reply:
This sgtm